### PR TITLE
Remove 1D approximation part from SHXF

### PR DESCRIPTION
**Pull request by jkha-unist**

### CODE REVIEW

File: src/mqc/shxf.py [hunk 1]
```diff
@@ -11,24 +11,14 @@ class Auxiliary_Molecule(object):
...
     """
-    def __init__(self, molecule, l_xf1d):
+    def __init__(self, molecule):
         # Initialize auxiliary molecule
-        if (l_xf1d):
 
-            self.nat = 1
-            self.ndim = 1
-            self.symbols = ['XX']
+        self.nat = molecule.nat_qm
```
Comment (inseonglee) [inline src/mqc/shxf.py:17]: Remove a blank line

Comment (jkha-unist) [reply src/mqc/shxf.py]: Done

**Comment by swmoon-unist** [comment]
LGTM

**Comment by DaehoHan** [comment]
Gaussian 09 tested. It works in the same way before.

**Pull request merged**